src/app/components/MarketPercentageBar.tsx

```diff
@@ -0,0 +1,21 @@
+const MarketPercentageBar = ({
```

Comment:
Use the Progress component from shadcn/ui
https://ui.shadcn.com/docs/components/progress